code/modules/mob/living/carbon/martian/martian.dm

```diff
@@ -128,3 +130,19 @@
 		stat = CONSCIOUS
 	else
 		health = maxHealth - getOxyLoss() - getFireLoss() - getBruteLoss() - getCloneLoss()
+
+/mob/living/carbon/martian/Stat()
+	..()
+
+	if(head && istype(head, /obj/item/clothing/head/helmet/space/martian))
+		var/obj/item/clothing/head/helmet/space/martian/fishbowl = head
+		if(fishbowl.tank && istype(fishbowl.tank, /obj/item/weapon/tank))
+			var/obj/item/weapon/tank/internal = fishbowl.tank
+			stat("Internal Atmosphere Info", internal.name)
+			stat("Tank Pressure", internal.air_contents.return_pressure())
+			stat("Distribution Pressure", internal.distribute_pressure)
+
+
+/mob/living/carbon/martian/Login()
+	..()
+	update_icons()
```

Comment:
Attempted to use this to try and get the HUD to update appropriately (Possessing a thingum when it was already holding something did not show what it was holding/wearing in the HUD slots) but has not worked